tests/manual/todo-list.md

```diff
@@ -0,0 +1,86 @@
+## Loading
+
+1. The data should be loaded with:
+  * two paragraphs,
+  * todo list with eight items, where 2,4 and 7 are checked
+  * two paragraphs,
+  * numbered list with one item,
+  * todo list with one unchecked item,
+  * bullet list with one item.
+2. Toolbar should have two buttons: for bullet and for numbered list.
```

Comment:
I do not get it. The toolbar has more than 2 buttons. It has also to-do list button.